javascript/javascript1/week3/script.js

```diff
@@ -16,10 +16,85 @@ function removeName(removalName) {
   for (let i = 0; i < names.length; i++) {
     if (names[i] === "Ahmad") {
       names.splice(i, 1);
+      break;
     }
   }
 }
 removeName("Ahmad");
 // Code done
 
 console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+// ######  When will we be there??
+
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function travelTimeCalc(travelInformation) {
+  return travelInformation.destinationDistance / travelInformation.speed;
+}
+
+const travelTime = travelTimeCalc(travelInformation);
+const travelHours = Math.floor(travelTime);
+const travelMinutes = travelTime - travelHours;
+
+console.log(travelTime); // 8 hours and 38 minutes
+
+// ### Series duration of my life
+
+const seriesDurations = [
+  {
+    title: "Lost",
+    days: 3,
+    hours: 18,
+    minutes: 0,
+  },
+  {
+    title: "Prison Break",
+    days: 2,
+    hours: 18,
+    minutes: 0,
+  },
+  {
+    title: "Breaking Bad",
+    days: 1,
+    hours: 22,
+    minutes: 29,
+  },
+];
+
+let averageLifeSpan = 80 * 365 * 24 * 60;
+let percentageOfTotal = 0;
+
+function logOutSeriesText() {
```

Comment:
You did a good job , everything looks nice. If you want to practice a bit more , you could try and make a function for what you did, from line 74 to 82.